ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -26,6 +27,12 @@ protected void configure() {
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
 		boolean enabled = true;
+		Optional<Long> lingersMs;
+		Optional<Integer> batchSize;
+		Optional<Integer> sendBufferBytes;
+		Optional<Integer> maxInFlightRequestsPerConnection;
+		Optional<Long> bufferMemory;
```

Comment:
Why are these all optionals does kafka handle these special? There dosent seem to be any use of it internally.